### PR TITLE
R2CON2019 Fix Deleting old Types before Load

### PR DESCRIPTION
**Pull request by thestr4ng3r**
When re-loading multiple types in a single file, only one was deleted before loading the new ones, here is an example: https://github.com/radare/radare2-regressions/pull/1945

### CODE REVIEW

File: libr/anal/type.c [hunk 1]
```diff
@@ -95,21 +95,28 @@ R_API void r_anal_remove_parsed_type(RAnal *anal, const char *name) {
...
-			// Now add the type to sdb.
-			sdb_query_lines (anal->sdb_types, parsed);
-			save_parsed_type_size (anal, parsed);
 		}
 		free (type);
 	}
+
+	// Now add the type to sdb.
+	sdb_query_lines (anal->sdb_types, parsed);
+	save_parsed_type_size (anal, parsed);
```
Comment (radare) [inline libr/anal/type.c]: If this is an internal function shoukd start with __

Comment (thestr4ng3r) [reply libr/anal/type.c]: Which function? `save_parsed_type_size`? Since when do we prefix our internal functions with __?

Comment (radare) [reply libr/anal/type.c]: yes , for a while, see panels.c for example, or my dev slides

Comment (thestr4ng3r) [reply libr/anal/type.c]: k fixed

**Pull request merged**